.ci-builds/.azure-pipelines.yml

```diff
@@ -582,3 +582,44 @@ jobs:
       env:
         CC: gcc
         CXX: g++
+#-------------------------------------------------------------------------
+- job: ICU4C_icuwriteuprops_Ubuntu_1804
+  displayName: 'C: Create Artifacts from icuwriteuprops'
+  timeoutInMinutes: 30
+  pool:
+    vmImage: 'ubuntu-18.04'
+  steps:
+    - checkout: self
+      lfs: true
+      fetchDepth: 10
+    - script: |
+        cd icu4c/source && ./runConfigureICU Linux && make -j2
+      displayName: 'Build'
+      env:
+        CC: clang
+        CXX: clang++
+    - script: |
+        mkdir icu4c/source/full_out
+        cd icu4c/source && LD_LIBRARY_PATH=lib ./bin/icuwriteuprops -A -I -c -d full_out
+        zip -rj icuwriteuprops_full.zip full_out/
```

Comment:
I picked the filename arbitrarily.  Do you think it should be something else, perhaps with the ICU version in it?

Reply:
how about uprops_full.zip ? the "icuwrite" prefix is good for the tool but weird for its output.

Reply:
Hmm, I was just thinking that we may want to add additional data that is not just unicode properties (e.g., collation or segmentation data).  So maybe "structures_full.zip" ?

Reply:
I would much prefer separate uprops_full.zip vs. collation_data.zip etc.